### PR TITLE
Config option for item + warning message, cleaned up code, added a command to toggle phone, fixed twitter icon and redesigned Dark Chat.

### PR DESCRIPTION
**Pull request by RockySouthpaw**
Lines 744-759 were also removed as they contained commented out code that was already implemented <10-15 lines above.

I attached the config option for phone as an item to `function TooglePhone()` so the function can be called for independent of just the initial opening of the phone via keybind/command and still retain the config settings. This should prevent people from calling someone who doesn't have a phone and them being able to answer it, but it doesn't stop the person receiving the call.

If it was implemented after `if IsControlJustPressed(1, Config.KeyOpenClose) then` then it would of been done improperly and lead to the above scenario as described.

I included the warning message for all languages in the locales translated with google translate. I also edited the server.lua for both esx 1.1 and 1,2. I can only test for 1.1 but 1.2 should work fine as well.

I withheld my initial desire to properly spell toggle and left it as is.

### CODE REVIEW

File: resources/gcphone/client/client.lua [hunk 1]
```diff
@@ -715,19 +708,38 @@ RegisterNUICallback('deleteALL', function(data, cb)
...
-  menuIsOpen = not menuIsOpen
-  SendNUIMessage({show = menuIsOpen})
-  if menuIsOpen == true then 
-    PhonePlayIn()
-    TriggerEvent('gcPhone:setMenuStatus', true)
-  else
-    PhonePlayOut()
-    TriggerEvent('gcPhone:setMenuStatus', false)
+function TooglePhone()
+  if Config.ItemRequired == true then -- If
```
Comment (userMacieG) [inline resources/gcphone/client/client.lua:712]: You can simplify this to `if Config.ItemRequired == true then`

File: resources/gcphone/client/client.lua [hunk 2]
```diff
@@ -715,19 +708,38 @@ RegisterNUICallback('deleteALL', function(data, cb)
...
+    hasPhone(function (hasPhone)
+      if hasPhone == true then
+        print("hasphone")
```
Comment (userMacieG) [inline resources/gcphone/client/client.lua:715]: Delete

File: resources/gcphone/client/client.lua [hunk 3]
```diff
@@ -715,19 +708,38 @@ RegisterNUICallback('deleteALL', function(data, cb)
...
+        else
+          PhonePlayOut()
+          TriggerEvent('gcPhone:setMenuStatus', false)
+        end
+      elseif hasPhone == false and Config.NoPhoneWarning == true then
+        NoPhone()
+      end
+    end)
+  elseif Config.ItemRequired == false then
+    print("config false")
```
Comment (userMacieG) [inline resources/gcphone/client/client.lua:730]: Delete

File: resources/server 1-1/server.lua [hunk 4]
```diff
@@ -31,19 +31,18 @@ end
...
-        if items == nil then
-            cb(0)
-        else
-            cb(items.count)
-        end
-    end)
---]]
+
+ESX.RegisterServerCallback('gcphone:getItemAmount', function(source, cb, item)
+    print('gcphone:getItemAmount call item : ' .. item)
```
Comment (userMacieG) [inline resources/server 1-1/server.lua:36]: Delete

File: resources/server 1-2/server.lua [hunk 5]
```diff
@@ -30,7 +30,7 @@ end
   Un solution ESC basé sur la solution donnée par HalCroves
   https://forum.fivem.net/t/tutorial-for-gcphone-with-call-and-job-message-other/177904
 --]]
---[[
+
  ESX.RegisterServerCallback('gcphone:getItemAmount', function(source, cb, item)
 	print('gcphone:getItemAmount call item : ' .. item)
```
Comment (userMacieG) [inline resources/server 1-2/server.lua:35]: Delete

File: resources/gcphone/client/client.lua (hunk 1 above)
Comment (RockySouthpaw) [reply resources/gcphone/client/client.lua:712]: Thats neat. I generally always do == true but will use this from now

**Pull request merged**